src/AppMenuSearch.cc

```
@@ -0,0 +1,407 @@
+            if (searchMenu) {
+                searchMenu->hide();
+            }
+        });
+        m_searchMenu->addAction(newAction);
+    }
+
+    m_searchMenu->setUpdatesEnabled(true);
+    Q_EMIT repositionRequested();
+}
+
+void AppMenuSearch::clear()
```

Clang (14.0.6)

[warning] method 'clear' can be made static

(readability-convert-member-functions-to-static)